### DISCUSSION TITLE
Simultaneous Users in GitHub

### DISCUSSION THREAD

**Discussion by AlanSalgado** [community/community#67841] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How many simultaneous users can the GitHub website support?

**Comment by minhkhacode** [comment]
These rate limits must not interfere with the lawful use of the GitHub Site. If your request triggers rate limiting, you'll receive a response containing HTTP status code 429 along with informative HTML content.

**Comment by ghostinhershell** [comment]
Hey @AlanSalgado, thanks so much for reaching out here. Can you clarify your question a bit here? Are you asking about potentially [managing multiple accounts when using GitHub.com](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/managing-multiple-accounts)?
